src/blocks/stars/stars.css

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
```

Comment:
The use of `:nth-child(-n + x)` is correct for selecting the first `x` children, but ensure that the number of stars in the HTML matches the expected number for each modifier class (e.g., `stars--1`, `stars--2`, etc.). If the number of stars is dynamic, this approach might not work as intended.